src/index.ts

```diff
@@ -0,0 +1,19 @@
+const { GraphQLServer } = require('graphql-yoga')
+
+const typeDefs = `
+type Query {
+  info: String!
+}
+`
+
+const resolvers = {
+  Query: {
+    info: () => `This is the API of a Hackernews Clone`
```

Comment:
Pra vc não se perder numa tarefa futura que você vai enfrentar, poderia mudar essa query para `Hello`. Ela deve receber um parâmetro opcional `name` (uma string) e retornar `Hello, nome_passado!`. Se o parâmetro não for passado, ela retorna `Hello, world!`. Pode ser?